transforms/universal/doc_id/python/src/doc_id_transform_python.py

```diff
@@ -0,0 +1,71 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+from argparse import ArgumentParser, Namespace
+from typing import Any
+
+import pyarrow as pa
+import ray
+from data_processing.data_access import DataAccessFactoryBase
+from data_processing.runtime.pure_python.runtime_configuration import (
+    PythonTransformRuntimeConfiguration,
+)
+from data_processing.transform import AbstractTableTransform, TransformConfiguration
+from data_processing.utils import CLIArgumentProvider, TransformUtils
+from data_processing_ray.runtime.ray import (
+    DefaultRayTransformRuntime,
+    RayTransformLauncher,
+)
+from data_processing_ray.runtime.ray.runtime_configuration import (
+    RayTransformRuntimeConfiguration,
+)
+from doc_id_transform_base import IDGenerator, DocIDTransformBase, DocIDTransformConfigurationBase
+
+
+class DocIDPythonTransform(DocIDTransformBase):
```

Comment:
None of our pure python transform classes contain `Python`.  For consistency, can you rename w/o `Python`?

Reply:
Yes, because they are not derived from base. Only this one and ededup do

Reply:
By convention the python transforms run in the python runtime do not include `Python` in the name.  Consistency is key when we have so many transforms to manage.  Just because there is a common base class does not mean we need to violate the convention.